requirements.txt

```diff
@@ -26,9 +26,9 @@ requests==2.32.3
 tenacity==9.0.0
 testimony==2.4.0
 wait-for==1.2.0
-wrapanapi
+wrapanapi==3.6.6
 
 # Get airgun, nailgun and upgrade from master
-airgun @ git+https://github.com/SatelliteQE/airgun.git@master#egg=airgun
+airgun @ git+https://github.com/SatelliteQE/airgun.git@00075a209e24df0e77c18b484631485bfc10f54f#egg=airgun
```

Comment:
Hmm, I don't understand this fix. Won't it block us from using latest code?

Reply:
Sure. But the latest code doesn't install, which mean's right now, we can't use it anyway.

(We can also go an revert that selenium bump in airgun, and then implement CI in airgun that checks that robottelo still works after an airgun change)

Reply:
So this is a temporary fix? The second option sounds like a lot of work but a proper fix.

Reply:
Lets hold the selenium bump in airgun. If we limit ourselves for airgun here then all the other fixes made in airgun wont be applied.

Reply:
IMHO yes

Reply:
@jyejare will you be fixing this issue then?

Reply:
Right, please don't solve this by pinning airgun version, it will stop us from developing all UI tests! The corect way is to fix/update dependencies in a way that we can use current versions and avoid this necromancy completely.

Reply:
Looking at the deptree, the reason we are using such old versions is wrapanapi requiring `openshift==0.3.4` while current version is 0.13.2. There might be more dragons but this is the first I've met.

Reply:
Correct, that's what I've written in the problem description.

Reply:
Oh. Right. We've met the same dragon.

Reply:
Is this time to start digging in wrapanapi?

Reply:
Probably depends on how many pitchforks and shovels you have on a Friday.

Reverting that airgun commit (and ensuring dependabot doesn't recreate that mess again) is probably a good enough fix for a Friday.

And then we can go and see how to fix wrapanapi to be compatible with dependencies released after 2017 ;-)

Reply:
Agree :)

Reply:
So this is a quick fix I made. It's a long time since I've played with dependencies and dependabot, please check if I didn't screw up something.
https://github.com/SatelliteQE/airgun/pull/1668

Reply:
Sounds like a plan. Acked, but I'm not yet high ranking enough to merge it:)